### PR TITLE
New release of pyml 20171113

### PR DESCRIPTION
**Pull request by thierry-martinez**

### CODE REVIEW

**Comment by gasche** [comment]
The CI fails on <4.02 OCaml versions. If you don't intend to support them, you should use an `available` bound. Otherwise, I suppose this is because you introduced uses of the `Bytes` package, you may be able to keep supporting older version through a bytes-compatibility package.

File: packages/pyml/pyml.20171102/opam [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+homepage: "http://pyml.gforge.inria.fr"
+bug-reports: "http://pyml.gforge.inria.fr/tracker"
+license: "BSD"
+dev-repo: "https://scm.gforge.inria.fr/anonscm/git/pyml/pyml.git"
+build: [make "all" "pymltop" "pymlutop" {utop:installed} "PREFIX=%{prefix}%"]
+install: [make "install" "PREFIX=%{prefix}%"]
+remove: [make "uninstall" "PREFIX=%{prefix}%"]
+depends: ["ocamlfind" {build}]
+depopts: ["utop"]
+version: "20171102"
```
Comment (avsm) [inline packages/pyml/pyml.20171102/opam]: This needs a line : `available: [ ocaml-version >= "4.02.0" ]` and it should be good merge

**Comment by gasche** [comment]
@thierry-martinez: gentle ping, there is not much left to do here.

**Comment by thierry-martinez** [comment]
Sorry for the late answer. I wish that pyml could work for OCaml <4.02.0, indeed: the problem was that I missed that there were some changes in the Bigarray C API between 4.01.0 and 4.02.0. I also see that it is probably a bad idea to have date-based versions with OPAM : I don't understand why CI tries to install pyml.20171102 while the version is now pyml.20171113. Should I open a new pull request for this version?

File: packages/pyml/pyml.20171113/opam [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+homepage: "http://pyml.gforge.inria.fr"
+bug-reports: "http://pyml.gforge.inria.fr/tracker"
+license: "BSD"
+dev-repo: "https://scm.gforge.inria.fr/anonscm/git/pyml/pyml.git"
+build: [make "all" "pymltop" "pymlutop" {utop:installed} "PREFIX=%{prefix}%"]
+install: [make "install" "PREFIX=%{prefix}%"]
+remove: [make "uninstall" "PREFIX=%{prefix}%"]
+depends: ["ocamlfind" {build}]
+depopts: ["utop"]
+version: "20171113"
```
Comment (rgrinberg) [inline packages/pyml/pyml.20171113/opam]: I don't think there's a need to include a `version:` line here. However, you should include a minimum OCaml version (and possibly a maximum one). It's very unusual for packages to support OCaml pre 4.02 for example.

File: packages/pyml/pyml.20171113/opam [hunk 3]
```diff
@@ -0,0 +1,14 @@
+opam-version: "1.2"
+name: "pyml"
```
Comment (rgrinberg) [inline packages/pyml/pyml.20171113/opam]: There's no need for this `name:` line.

**Comment by thierry-martinez** [comment]
@rgrinberg I removed the version flag, thank you. `pyml` is supposed to be compatible with OCaml from 3.12.0. I just published a new version using `stdcompat`, with a new pull request. I close this one, thus. Thank you again!

**Comment by rgrinberg** [comment]
@thierry-martinez NP. It's good to set the minimum version anyways however. `3.12` in your case.

**Pull request closed without merging**